lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -3570,4 +3570,9 @@ public function getStringLiteralQuoteCharacter()
     {
         return "'";
     }
+
+    public function getSequenceDataSQL($sequenceName, $schemaName)
```

Comment:
DBAL requires PHP 7.1, please add parameter and return types where applicable.

Reply:
It should probably be renamed to `getSequenceMetadataSQL`, `data` is not explicit.